app/deploy/syndesis.yml

```diff
@@ -1145,6 +1149,26 @@ objects:
             - targets:
               - localhost:9090
 
+          metric_relabel_configs:
```

Comment:
@dhirajsb I guess this comes in from one of your latest PRs. Don't forget to run 'generator/run.sh' when you change the templates (that what I did to get my changes in, but it picked up yours as well)

Reply:
I thought I did, must have gotten overwritten somehow.